homeassistant/components/hausbus/event_handler.py

```diff
@@ -0,0 +1,20 @@
+"""Event handler abstract class."""
+
+from abc import ABC, abstractmethod
+from collections.abc import Callable, Coroutine
+from typing import Any
+
+from .entity import HausbusEntity
+
+
+class IEventHandler(ABC):
```

Comment:
Prefixing interface classes with `I` looks like a Java thing, we don't do that in this project, I suggest to change to:
```suggestion
class EventHandler(ABC):
```

OK to keep if you want to be consistent with the pyhausbus library though